std/string/string_utils.zig

```diff
@@ -0,0 +1,169 @@
+const std = @import("../index.zig");
+const mem = std.mem;
+const math = std.math;
+const Set = std.BufSet;
+const debug = std.debug;
+const assert = debug.assert;
+const ascii = std.string.ascii;
+const utf8 = std.string.utf8;
+
+/// Returns an iterator that iterates over the slices of `buffer` that are not
```

Comment:
these docs are incorrect

Reply:
The only incorrect line is that it references 'bytes' :)